seatsio-ios/SeatingChartConfig.swift

```diff
@@ -33,10 +33,9 @@ public class SeatingChartConfig: Encodable {
     var mode: String?
     var loading: String?
     var ticketListings: [TicketListing]?
-    var holdOnSelect: Bool?
     var holdOnSelectForGAs: Bool?
     var holdToken: String?
-    var regenerateHoldToken: Bool?
+    var session: String?
```

Comment:
We could also use an enum here.

Reply:
true, but we don't for other fields (`mode`, `colorScheme`), so I decided to keep it consistent